src/main/java/com/fasterxml/jackson/databind/deser/std/EnumDeserializer.java

```diff
@@ -77,14 +81,16 @@ public EnumDeserializer(EnumResolver byNameResolver, Boolean caseInsensitive)
     /**
      * @since 2.9
      */
-    protected EnumDeserializer(EnumDeserializer base, Boolean caseInsensitive)
```

Comment:
Will need to leave old constructor for backwards-compatibility; I can add it after merging.